paperslicer/metadata/resolver.py

```
@@ -0,0 +1,92 @@
+            parts = date.replace("/", "-").split("-")
+            try:
+                if len(parts) == 1 and parts[0].isdigit():
+                    out["date"] = parts[0]
+                elif len(parts) == 2 and all(p.isdigit() for p in parts):
+                    y, m = parts
+                    out["date"] = f"{int(y):04d}-{int(m):02d}"
+                elif len(parts) >= 3:
+                    y, m, d = parts[:3]
+                    out["date"] = f"{int(y):04d}-{int(m):02d}-{int(d):02d}"
+            except Exception:
+                pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)